app/scripts/metamask-controller.js

```diff
@@ -382,9 +382,11 @@ export default class MetamaskController extends EventEmitter {
       initState.PermissionsMetadata,
     );
 
-    this.workerController = new WebWorkerExecutionEnvironmentService({
+    this.workerController = new IframeExecutionEnvironmentService({
       setupPluginProvider: this.setupPluginProvider.bind(this),
-      workerUrl: WORKER_BLOB_URL,
+      iframeUrl: new URL(
+        'https://metamask.github.io/iframe-execution-environment/',
```

Comment:
We're probably going to need to figure out a different place to host this, but it's fine to in-line it for now.

Reply:
also, if this url was on IPFS I think it would be better when we get more production ready. Content addressable gives us nice benefits.